try {
  hello();
}catch(error) {
  console.log(erorr);
}

try {
  anotherFn();
} catch {
  console.log('Esto es un error!');
}

